lib/widgets-page.php

```diff
@@ -58,15 +58,21 @@ function gutenberg_widgets_init( $hook ) {
 	wp_add_inline_script(
 		'wp-edit-widgets',
 		sprintf(
-			'wp.editWidgets.initialize( "widgets-editor", %s );',
+			'window.addEventListener( "load", function() {
+				wp.editWidgets.initialize( "widgets-editor", %s );
```

Comment:
I think the theory here is that `editWidgets` is specific to the widgets screen and we didn't initially think of using it in the customizer directly.

I can see how it's a quick way to arrive to something usable, the issue though is that it forces us to play with CSS to have a sidebar with the "save" button... 

I think ideally, we'd:

 - Extract the common parts between the customizer and the widgets screen into a `@wordpress/widgets` or `@wordpress/widget-areas` package 
 - Build a new `widgets-customizer-panel` package that use the common parts in order to bootstrap the customizer package.
 - Try to merges the save button with the customizer's save button. (We shouldn't need two separate saving actions)

Reply:
What if we do that all in the same, current package. Merging the save buttons when rendering in the customizer should be simpler than splitting this into multiple packages.

Reply:
cc @jorgefilipecosta